package/package_esp8266com_index.template.json

```diff
@@ -55,6 +55,9 @@
             {
               "name": "WifInfo"
             }
```

Comment:
missing comma after the bracket above and an extra one at the end of "ESPDuino" brackets